public/locales/fr-FR/grafana.json

```diff
@@ -38,7 +38,7 @@
       "query-tab": "Requête",
       "stats-tab": "Statistiques",
       "subtitle": "{{queryCount}} requêtes avec un délai total de requête de {{formatted}}",
-      "title": "Inspecter : {{panelTitle}}"
+      "title": "Inspecter\u00a0: {{panelTitle}}"
```

Comment:
What happened to all these french translations?

Reply:
that's the output when running `yarn i18n:extract`... i asked @joshhunt and he said it was correct to commit it, crowdin will "fix" it later